Assets/Controllers/CameraMotionHandler.cs

```diff
@@ -79,6 +78,12 @@ public void SetMoveTargetPosition(int upDown, int leftRight)
         newPosition.z = newZ;
     }
 
+    private void GetNewXandNewZ(int upDown, int leftRight, out float newX, out float newZ)
```

Comment:
This is a step in right direction however I was getting at abstracting the logic below into one method. For example, you could have a method that is named NewPosition that would do an X or Z based on your modifier (the below is pseudo code, it may not run):

```
private void NewPosition(float axis, int movement, out float newAxis)
{
    newFloat = axis + (movement * cameraMoveSpeed * Time.deltaTime);
}
```
You would then call that function to produce your X and Z with out duplicating the formula.